### ISSUE TITLE
Problem with booting CB2

### ISSUE THREAD

**Issue by WEBOWNIA** [bigtreetech/Manta-E3EZ#45]
I have an issue with booting CB2. I'm following: https://www.youtube.com/watch?v=XfZppjHU0ak

The difference is that I haven't connected a monitor because I don't yet have an HDMI adapter and am waiting for it to arrive. So, I can't see what's going on there...

I'm expecting to see the device name on the list of connected devices, but nothing appears, so I'm assuming either Klipper isn't starting for some unknown reason, or maybe I'm doing something wrong.

**Comment by WEBOWNIA** [comment]
Is there any way to check if the board and CB2 are not damaged?

I should add that I bought new units. I connected and fastened them with small M2 screws according to the instructions.

I connected the WiFi antenna.

I plugged in a 24V power supply, as that’s also recommended.

I also tried connecting via USB OTG cable by switching the DPDT switch and bridging jumpers 3 (USBOTG) and 4 (RPIBOOT) to flash using RKDevTool. Unfortunately, the computer doesn’t detect anything. I also tried uninstalling and reinstalling the included USB drivers. (Windows 10 Pro)

I’ll add that when I apply power, the red LED stays lit, so it seems like there’s some life in it.

![5cd82338-944f-4c53-8a13-90a5f889d8d7](https://github.com/user-attachments/assets/040a26a4-ba0a-4bf8-85d6-2aeb28044b81)
